src/doc/src/reference/unstable.md

```diff
@@ -338,26 +338,26 @@ This was stabilized in 1.79 in [#13608](https://github.com/rust-lang/cargo/pull/
 ### MSRV-aware resolver
 
 `-Zmsrv-policy` allows access to an MSRV-aware resolver which can be enabled with:
-- `resolver.something-like-precedence` config field
+- `resolver.incompatible-rust-versions` config field
 - `workspace.resolver = "3"` / `package.resolver = "3"`
 - `package.edition = "2024"` (only in workspace root)
 
 The resolver will prefer dependencies with a `package.rust-version` that is the same or older than your project's MSRV.
 Your project's MSRV is determined by taking the lowest `package.rust-version` set among your workspace members.
 If there is none set, your toolchain version will be used with the intent to pick up the version from rustup's `rust-toolchain.toml`, if present.
 
-#### `resolver.something-like-precedence`
+#### `resolver.incompatible-rust-versions`
 * Type: string
-* Default: "something-like-maximum"
-* Environment: `CARGO_RESOLVER_SOMETHING_LIKE_PRECEDENCE`
+* Default: "allow"
+* Environment: `CARGO_RESOLVER_INCOMPATIBLE_RUST_VERSIONS`
 
-Select which policy should be used when resolving dependencies.  Values include
-- `something-like-maximum`: prefer highest compatible versions of a package
-- `something-like-rust-version`: prefer versions of packages compatible with your project's Rust version
+Select how packages with incompatible rust-versions should be resolved.  Values include:
```

Comment:
It is unclear what an incompatible rust-version is. Should we link "incompatible rust-versions" to `package.rust-version` in the manifest?

Reply:
I've called out the field name.  We should eventually turn all of those into links.